http_logs/challenges/common/default-schedule.json

```diff
@@ -70,9 +70,9 @@
         },
         {
           "operation": "term",
-          "warmup-iterations": 500,
-          "iterations": 100,
-          "target-throughput": 50
+          "warmup-iterations": {%- if runtime_script_grok is defined %}50{% else %}500{% endif %},
+          "iterations": {%- if runtime_script_grok is defined %}10{% else %}100{% endif %},
+          "target-throughput": {%- if runtime_script_grok is defined %}0.125{% else %}50{% endif %}
```

Comment:
On my desktop this runs in about seven seconds so I think targeting 8 seconds in CI is a good start. We can tune it from there.